torch/_torch_docs.py

```diff
@@ -145,21 +145,21 @@ def merge_dicts(*dicts):
            r"""
 .. function:: add(input, other, out=None)
 
-Adds the scalar :attr:`other` to each element of the input :attr:`input`
-and returns a new resulting tensor.
+Each element of the tensor ``other`` is added to each element of the
```

Comment:
To be consistent with the rest of our documentation, ese should be `:attr:`other`` and `:attr:`input``